app/views/ems_container/_show_topology.html.haml

```diff
@@ -0,0 +1,84 @@
+.container_topology{'data-ng-app' => "topologyApp", 'ng-controller' => "containerTopologyController"}
```

Comment:
Is this a "copy paste" from the show.html.haml under the container_topology? 
There should be a better way to reuse/redirect than copy pasting files, otherwise any change must be remembered to be done twice, and that's not a good practice.